pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[project]
+requires-python = "==3.12.*"
+
+dependencies = [
+    # CDO python packages, from /python:
+    "pycdo @ file:///${PROJECT_ROOT}/python/pycdo",
```

Comment:
See [/python/pycdo/pyproject.toml](https://github.com/code-dot-org/code-dot-org/blob/seth/setup-pycall/python/pycdo/pyproject.toml), which includes deps on outside packages. These will be laddered up and included in the overall venv for code-dot-org.